src/app/views/infoblock/infoblock.html

```diff
@@ -0,0 +1,4 @@
+<div class="panel-content text-center">
+  <h2 class="{{ $ctrl.colorClass }}">{{ $ctrl.value }}</h2>
+  <p class="font-alt color-light text-uppercase">{{ $ctrl.title}}</p>
```

Comment:
The `$ctrl.title` and `$ctrl.value` should use the one time binding syntax, since they're static values and don't change once initially drawn.

https://docs.angularjs.org/guide/expression#one-time-binding